src/index.tsx

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
-import { createRoot } from 'react-dom/client';
-import App from './App/App';
-import ErrorBoundary from './ErrorBoundary/ErrorBoundary';
-
-import './style.css';
-
-const container = document.getElementById('root');
-const root = createRoot(container);
-root.render(
-  <ErrorBoundary>
-    <App />
-  </ErrorBoundary>,
-);
+import { createRoot } from 'react-dom/client';
+import App from './App/App';
+import ErrorBoundary from './ErrorBoundary/ErrorBoundary';
+
+import './style.css';
+
+const container = document.getElementById('root') as HTMLBodyElement;
```

Comment:
Will check.

Reply:
Changed HTMLBodyElement to HTMLDivElement.